sgp4/functions.py

```diff
@@ -5,6 +5,11 @@
 routines instead.
 
 """
+from datetime import datetime
+
+import pytz
```

Comment:
Most Python users, alas, don't have `pytz` installed, so this will raise an error. And I would prefer a library like this that's so simple to not require time zone importation, when most users don't need that.

Try this instead, it's how I solved the same problem in Skyfield:

    class UTC(tzinfo):
        'UTC'
        zero = timedelta(0)
        def utcoffset(self, dt):
            return self.zero
        def tzname(self, dt):
            return 'UTC'
        def dst(self, dt):
            return self.zero

    utc = UTC()

Reply:
OK, fair point. I found this solution that uses the native `datetime`. I will implement it without pytz but with the same local time-to-UTC functionality.

```
from datetime import datetime, timezone

    datetime_now: datetime = datetime.now()
    print(f"Time in Turkey: {datetime_now} {datetime_now.tzinfo}")

    datetime_now_utc = datetime_now.astimezone(timezone.utc)
    print(f"Time in UTC   : {datetime_now_utc} {datetime_now_utc.tzinfo}")
```